RadarSDKTests/RadarAPIHelperMock.h

```diff
@@ -6,9 +6,8 @@
 //
 
 #import <Foundation/Foundation.h>
-#import <RadarSDK/RadarSDK.h>
-
-#import "RadarAPIHelper.h"
+#import <RadarSDK.h>
+#import "../RadarSDK/RadarAPIHelper.h"
```

Comment:
Yeah, I don't like this change, either, but 1) it's limited to our internal unit tests, and 2) future unit tests will hopefully be written in Swift. :)